frontend/src/pages/Profile.js

```diff
@@ -0,0 +1,149 @@
+import '../styles/Profile.css';
+import React, { useEffect, useState } from "react";
+import { Button, Icon, Image, Grid, Container, Popup} from 'semantic-ui-react'
+import { loadProfile} from '../utils/api';
+
+function Profile() {
+  const [allValues, setAllValues] = useState([]);
+  const [name, setName] = useState();
+  const [year, setYear] = useState();
+  const [age, setAge] = useState();
+  const [gender, setGender] = useState();
+  const [major, setMajor] = useState();
+  const [email, setEmail] = useState();
+  const [housingType, setHousingType] = useState();
+  const [shortDesc, setShortDesc] = useState();
+  const [longDesc, setLongDesc] = useState();
+  const [numRoomates, setNumRoomates] = useState();
+  const [fbUsername, setfbUsername] = useState();
+  const [igUsername, setigUsername] = useState();
+  const [scUsername, setscUsername] = useState();
+ //callApi().then(text=> setText(text));
+ 
+ useEffect(() => {
+  const getText = async () => {
+  const data = await loadProfile("5fadf6bba3e3324e794a1141");
```

Comment:
I would suggest changing the variable name to something more specific, like profile